examples/web/src/components/controller_app.js

```diff
@@ -0,0 +1,78 @@
+/*
+ * The software contained in this file is free and unencumbered software
+ * released into the public domain. Anyone is free to use the software
+ * contained in this file as they choose, including incorporating it into
+ * proprietary software.
+ */
+
+import NavControl from './NavControl.js';
+import VideoPlayer from './VideoPlayer.js';
+import SelectInput from './SelectInput.js';
+import ElementsControl from './ElementsControl.js';
+import ChangeProperty from './ChangeProperty.js';
+import BusConsole from './BusConsole.js'
+import BusControl from './BusControl.js'
+import FooterControl from './FooterControl.js'
+import GstdControl from './GstdControl.js'
+
+var gstc = new GstdClient("http://" + sessionStorage.address, sessionStorage.port);
+
+if (document.getElementById("controller")) {
+    Vue.prototype.$datas = {
+        file: '',
+        videoElement: null,
+        paused: false,
+        gstc: gstc,
+        pipeName: "jsgstc_player",
+        defaultAddress: "127.0.0.1",
+        defaultPort: "5000",
+        componentKey: 0,
+        selected_element: null,
```

Comment:
What is your naming standard? Some are `mixedCamelCase` and others are `snake_case`